NEWS.md

```diff
@@ -609,5 +609,10 @@
 
 15. Thanks to @ssh352, Václav Tlapák, Cole Miller, András Svraka and Toby Dylan Hocking for reporting and bisecting a significant performance regression in dev. This was fixed before release thanks to a PR by Jan Gorecki, [#5463](https://github.com/Rdatatable/data.table/pull/5463). 
 
+16. `key(x) <- value` is now fully deprecated (from warning to error). Use `setkey()` to set a table's key. We started warning not to use this approach in 2012, with a stronger warning starting in 2019 (1.12.2). This function will be removed in the next release.
+
+17. Argument `logicalAsInt` to `fwrite()` now warns. Use `logical01` instead. We stated the intention to begin removing this option in 2018 (v1.11.0). It will be upgraded to an error in the next release before being removed in the subsequent release.
```

Comment:
Possibly introduces revdeps failures, and slows down release by asking them to amend.

Reply:
I disagree. Once again we should merge and check revdeps.

Moreover, in this case we can trivially check the entirety of GitHub for downstream usages, of which there are practically 0 -- most of the 41 results here are just {data.table} forks/copies:

https://github.com/search?q=lang%3AR%20%2FlogicalAsInt%5Cs*%3D%2F&type=code

Reply:
There are no (0) CRAN packages using `logicalAsInt`:

https://github.com/search?q=lang%3AR+%2FlogicalAsInt%5Cs*%3D%2F+org%3Acran&type=code